### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by rahulyadav139**
I have created the login page for this project. Kindly review it.

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
great UI, looks good to merge. 🚀

File: index.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.main .main__intro-subtext {
+  font-weight: 500;
+  margin-bottom: 2rem;
+}
+.main div {
+  flex-basis: 50%;
+}
+
+@media only screen and (max-width: 414px) {
```
Comment (PuneetP16) [inline index.css:35]: Can use variables instead of values in max-width. It would help in managing media queries if used in multiple pages.

**Review by PuneetP16 — APPROVED** [review]
Improve the contrast ratio of login text and signup text.
image

File: index.css (hunk 1 above)
Comment (rahulyadav139) [reply index.css:35]: I think px is fine here. For contrast ratio, I need to make required changes in the component library colour css file.

**Pull request merged**